src/include/wt_internal.h

```diff
@@ -379,6 +385,9 @@ typedef uint64_t wt_timestamp_t;
 #include "block.h"
 #include "bloom.h"
 #include "btmem.h"
+#include "bt_traits.h"
+#include "cache_las_traits.h"
+#include "rec_traits.h"
```

Comment:
Although I agree with the comments of others on the file naming and moving things out of the traits files, if this sticks, these need to be alphabetized.